src/main/kotlin/com/launchdarkly/intellij/action/ChangeFallthroughAction.kt

```diff
@@ -51,7 +51,8 @@ class ChangeFallthroughAction : AnAction {
         val project = event.project ?: return
         val currentComponent = event?.inputEvent?.component ?: return
         val selectedNode =
-            project.service<FlagToolWindow>().getPanel().tree.lastSelectedPathComponent as DefaultMutableTreeNode
+            project.service<FlagToolWindow>().getPanel()
+                .getFlagPanel().tree.lastSelectedPathComponent as DefaultMutableTreeNode
```

Comment:
It looks like this same selectedNode logic gets used quite a lot, in 7 places. Is it possible to DRY it like so?

```java
    fun getSelectedNode(event: AnActionEvent): DefaultMutableTreeNode {
        return event.project!!.service<FlagToolWindow>().getPanel()
            .getFlagPanel().tree.lastSelectedPathComponent as DefaultMutableTreeNode
    }

    // then other functions can call it like so
    override fun actionPerformed(event: AnActionEvent) {
        val project = event.project ?: return
        val selectedNode = getSelectedNode(event)

```

Reply:
Sounds good to me, created https://app.shortcut.com/launchdarkly/story/145418 to track

Reply:
I'm going to push for this to be done now rather than tech debting this to another ticket because it's easy enough to do.

Reply:
It will be quick, but it should be it's own piece of work and will be a smaller cleaner PR separate from this.